### PR TITLE
UnderdogMedia - CDN on a cookieless domain. Optional divId param. Pass bids to library as an array.

### PR DESCRIPTION
**Pull request by ojotoxy**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change

1. Move our header bidding library to a cookieless domain. This is for performance.
2. Pass bid requests to the library as an array, this allows performance enhancements within our library.
3. Add divId as an optional parameter to bidrequests. This can be used for reporting purposes.

### CODE REVIEW

**Comment by protonate** [comment]
Thanks for the PR, assigned for review.

**Comment by ojotoxy** [comment]
Added another optional parameter, subId. For reporting purposes.

File: test/spec/adapters/underdogmedia_spec.js [hunk 1]
```diff
@@ -40,8 +40,19 @@ describe('underdog media adapter test', () => {
 
         }
       };
-    }
+    },
+
+    BidRequestArray: function(arr){
+      return {
+        send: function(){
+          for(var i = 0; i < arr.length; i++){
```
Comment (jaiminpanchal27) [inline test/spec/adapters/underdogmedia_spec.js]: Please use Array.forEach instead of for loop.

**Comment by jaiminpanchal27** [comment]
@ojotoxy I have requested small change. Please review and update.

**Comment by jaiminpanchal27** [comment]
@ojotoxy Thanks. Looks good. Merging

**Pull request merged**